client/app/components/navigation/Tiles.tsx

```diff
@@ -88,46 +115,56 @@ export default function Tiles({
   }
 
   return (
-    <>
-      <Grid container>
-        {contents &&
-          contents.map((content, index) => (
-            <Grid
-              key={index}
-              item
-              component={Card}
-              xs={12}
-              sm={6}
-              md={3}
-              sx={{
-                margin: 6,
-                display: "flex",
-                flexDirection: "column",
-                justifyContent: "space-between",
-              }}
-            >
-              <CardContent data-testid="dashboard-nav-card">
-                <Typography variant="h5" component="h2">
-                  {content.title}
-                </Typography>
```

Comment:
Split on whether we should use MUI or regular HTML elements with Tailwind for basic html/css stuff. I ran into some issues with MUI Grid and switched this to plain html/Tailwind but happy to go back to MUI if the team finds it easier to use.

Reply:
I like tailwind.  Also, I had issues with some MUI components in the Jest testing.